packages/calling-component-bindings/src/errorBarSelector.ts

```diff
@@ -65,9 +65,15 @@ export const errorBarSelector: ErrorBarSelector = createSelector(
     if (diagnostics?.media.latest.microphonePermissionDenied) {
       activeErrorMessages.push({ type: 'callMacOsMicrophoneAccessDenied' });
     }
-    if (diagnostics?.media.latest.cameraFreeze) {
-      activeErrorMessages.push({ type: 'callCameraAlreadyInUse' });
+
+    if (deviceManager.deviceAccess?.video === false) {
+      activeErrorMessages.push({ type: 'callCameraAccessDenied' });
+    } else {
```

Comment:
nit: else if (,,,) { ... } instead of else { if (...) { ... } }

Reply:
Thanks.